src/version/version.controller.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import {
+  Controller,
+  Get,
+  HttpStatus,
+  Post,
+  Query,
+  UseGuards,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { ApiExcludeEndpoint, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { ApiKeyGuard } from '../auth/guards/api-key.guard';
+import { VersionDto } from './dto/version.dto';
+import { VersionService } from './version.service';
+
+@ApiTags('Version')
+@Controller('version')
+export class VersionController {
+  constructor(private readonly versionService: VersionService) {}
+
+  @ApiOperation({ summary: 'Gets the version of the Iron Fish API' })
+  @Get()
+  async version(): Promise<string> {
+    const latestVersion = await this.versionService.getLatest();
+    return latestVersion ? latestVersion.version : '';
```

Comment:
I'll let you make the call here and don't have a strong opinion, but I prefer to return objects (even if it's a single field). That way we can add more fields easily (if we ever want to return the release date for example), deprecate old fields, etc.

Reply:
Yeah, I think you're right. Originally I thought to keep it simple as possible, but returning a string is pretty non-standard and pigeonholes future updates quite a bit (or breaking changes). Good call.